### ISSUE TITLE
module 'base64' has no attribute 'encodestring' on Python 3.9

### ISSUE THREAD

**Issue by laanwj** [gkrizek/ghi#26]
Sending to IRC fails with this message:
```
Feb 20 14:03:09 … python3[399]: IRC - Connecting to irc.libera.chat:6697 with nick bitcoin-git and channels: #bitcoin-commits,#bitcoin-core-dev
Feb 20 14:03:09 … python3[399]: IRC - There was a problem sending messages to IRC
Feb 20 14:03:09 … python3[399]: IRC - module 'base64' has no attribute 'encodestring'
```
According to upstream [issue 39351](https://bugs.python.org/issue39351), this function was deprecated since Python 3.1 in favor of `base64.encodebytes` and `decodebytes`.

**Comment by laanwj** [comment]
Edit: oh, already fixed in 8ab6343d3d03f6eeee8003f082d7760e53f8a047 / #15, just needed to pull :smile:

**Comment by gkrizek** [comment]
@laanwj That's correct! And I just cut a new release with these fixes:

https://github.com/gkrizek/ghi/releases/tag/v0.2.0
